walkthroughs/3a-low-code-api-development/walkthrough.adoc

```diff
@@ -0,0 +1,465 @@
++
+[subs="attributes+"]
+----
+https://wt-{user-sanitized-username}-3scale.{openshift-app-host}
+----
++
+If the wildcard proxy is enabled in your cluster, this route points to the wildcard APIcast URL. Otherwise, the route points to the shared staging APIcast URL for the *3scale* project in OpenShift
+
+. In the *MAPPING RULES* section, add a *POST* mapping with the pattern `/greeting`.
+
+.. Select *Update & test in Staging Environment*
+
+[NOTE]
+====
+If you have previously used the ```https://wt-{user-sanitized-username}-3scale.{openshift-app-host}``` route, that service is no longer available.
+For example, if you have completed the *Managing and protecting APIs using API management* walkthrough, that API is no longer available because you have reused the route.
+====
+
+
+[type=verification]
+Is the API service available?
+You might encounter a *403: Authentication failed* message. You can ignore this message, the issue is resolved in a later step.
+
+[type=verificationFail]
+Verify that you followed each step in the procedure above.  If you are still having issues, contact your administrator.
+
+=== Configuring your API
+
+. Create a new *Application Plan*:
+.. Select *Applications > Application Plans* from the side navigation.
+.. Select *Create Application Plan*.
+.. Enter the following for *Name* and *System name*:
++
+[subs="attributes+"]
+----
+low-code
+
+----
+.. Leave the other fields with their default values.
+.. Select *Create Application Plan*. You will be redirected to the *Application Plans* screen.
+.. Select the *Publish* button, beside your plan list item, to publish the Plan.
+
+. Create a new *Application* for the *Developer* Group, assigned to the Plan:
+.. Select *Audience* from the top navigation bar dropdown.
+.. Select the *Developer* Account to open the *Account Summary* page.
+.. Select the *(num) Application* item from the breadcrumb to view Applications.
+.. Select the *Create Application* button in the top right.
+.. Select the *low-code* Plan in the *Application plan* dropdown.
+.. Enter the following for *Name* and *Description*:
++
+[subs="attributes+"]
+----
+low-code-app
+----
+.. Select *Create Application*.
+
+. Set a custom *User Key* for the application:
+.. On the *low-code* application screen you were redirected to, scroll to the *API Credentials* section.
+.. Click the green pencil icon beside the *API User Key*
+.. In the *Set Custom User Key* modal dialog, enter:
++
+[subs="attributes+"]
+----
+test
+----
+.. Select *Set Custom Key*.
+
+
+[type=verification]
+****
+Use an HTTP client to invoke the route, for example:
+
+----
+curl -d '{"name":"John"}' -H "Content-Type: application/json" -X POST "{route}/greeting?user_key=test"
+
+----
+
+Did the message "Hello from John" appear in your Slack channel?
+****
+
+[type=verificationFail]
+Verify that you followed each step in the procedure above.  If you are still having issues, contact your administrator.
+
+=== Invoking the API
```

Comment:
This section seems to duplicate the previous one:

![Screenshot from 2019-08-22 15-22-57](https://user-images.githubusercontent.com/12085451/63522725-d36f6980-c4f0-11e9-814e-3d4f0645c3d8.png)